config.yml

```diff
@@ -1686,6 +1686,20 @@ nodes:
 
           return 1
           ^^^^^^^^
+  - name: ScopeNode
+    child_nodes:
+      - name: parameters
+        type: node?
+        kind: ParametersNode
+      - name: statements
+        type: node?
+        kind: StatementsNode
```

Comment:
What's the right type here, then? I haven't implemented rescues in the compiler yet, I'm not sure if they compile totally separately to the body or not

Reply:
I would imagine you would leave off `kind:` here and handle it in the compiler. In that way it would be a `yp_node_t *` which you would type check and then cast.